src/Process/LayeredMaterialNodeProcessing.js

```diff
@@ -202,13 +203,13 @@ export function updateLayeredMaterialNodeImagery(context, layer, node) {
     material.setLayerVisibility(layerIndex, layer.visible);
     material.setLayerOpacity(layerIndex, layer.opacity);
 
+    const failureParams = node.layerUpdateState[layer.id].failureParams;
     const ts = Date.now();
     // An update is pending / or impossible -> abort
-    if (!layer.visible || !node.layerUpdateState[layer.id].canTryUpdate(ts)) {
+    if (!layer.visible || (!node.layerUpdateState[layer.id].canTryUpdate(ts) && !failureParams)) {
```

Comment:
This condition is becoming hard to parse.
If I understand correctly, you want to skip update if:
- layer is not visible
- layer is in error and failureParam is unset?

Why would you want to update if `canTryUpdate` returns false? (remember than `layerUpdateState` spaces retries so I don't think it makes sense to ignore it simply because we have `failureParams` from a previous attempt.

Reply:
the reason is if there was an error and we want to re-fetch with a new target level there are two things that go against new try

* the max number of test that can block a try with a new target level
* the pause time between tests which is also blocking

I was thinking of adding a status of the kind : re-try with new Params